src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -107,8 +107,23 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
                 ecommerceSetOrderShipping(input: $input) {
                   orderOrError {
                     ... on OrderWithMutationSuccess {
+                      __typename
                       order {
+                        id
                         state
+                        requestedFulfillment {
+                          __typename
+                          ... on Ship {
+                            name
+                            addressLine1
+                            addressLine2
+                            city
+                            region
+                            country
+                            postalCode
+                            phoneNumber
+                          }
+                        }
```

Comment:
FYI Extracting and sharing a fragment like this with that of the container is what unmasking is meant for.